app.js

```diff
@@ -130,7 +130,7 @@ passport.deserializeUser(function(user, done) {
 passport.use(new LocalStrategy(function(username, password, done) {
     process.nextTick(function() {
         app.db.collection(userDB).findOne(
-            {'username': username },
+            {'SEARCH': username.toUpperCase() },
```

Comment:
if we could have search in lower case that would be better. the game templates already have a search parameter and it is in lowercase.